apps/server/tests/metrics.test.ts

```
@@ -787,52 +815,61 @@ describe("metrics", () => {
+        wg_variables: JSON.stringify({ startDate: START_DATE })
+      });
 
     // Fetch the Metric record for the same date
-    const records = result.data;
-    const record = records?.filter(record => record.date === START_DATE)[0];
+    const records = response.body.data;
+    const record = records?.filter((record: any) => record.date === START_DATE)[0];
 
     expect(record).not.toBeNull();
     // The calculation logic of the liquid backing and backed supply metrics are already tested above, so this just checks that the derived metric is correct
     expect(record?.treasuryLiquidBackingPerOhmBacked).toBeCloseTo(record?.treasuryLiquidBacking! / record?.ohmBackedSupply!);
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)